mitodl/micromasters
# Fixed various profile field validations for exams (#2804)

I addressed a few other small issues I noticed while in the code, this is noted in section 2 below.

#### What are the relevant tickets?
Fixes #2804 

#### What's this PR do?

* Added length validation to all name fields
* Added CP-1252 validation where it was missing (city).
* Normalized first name -> given name, last name -> family name language

#### How should this be manually tested?
Go to the profile page and enter first names that are longer than 30 characters and long names that are longer than 50 characters and verify you see the error message. Also verify you can't enter non-latin characters in the city since I fixed that too while I was in there.

#### Screenshots (if appropriate)
![selection_039](https://cloud.githubusercontent.com/assets/28598/23724881/73ee6ad4-041c-11e7-94c7-1407f38b21ef.png)


#### What GIF best describes this PR or how it makes you feel?
![validating](http://media4.giphy.com/media/P5E3zUUOimC9a/200w.gif)
